common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -4034,7 +4034,192 @@ change_undead_demesne_provinces_religion_to_root_effect = {
 		religion = ROOT
```

Comment:
Got my 5th mutation, still not a n'raqi.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80706638-abadb800-8af9-11ea-8a1c-5353c6982350.png)
![image](https://user-images.githubusercontent.com/46384992/80706745-e152a100-8af9-11ea-8bf0-904023e488fe.png)

</details>

Reply:
youre supposed to get it when gaining 6th mutation